cinema/views.py

```diff
@@ -101,8 +105,27 @@
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "create":
+            return MovieCreateSerializer
+
+        if self.action == "upload_image":
+            return MovieImageSerializer
```

Comment:
The `get_serializer_class` method in `MovieViewSet` returns `MovieImageSerializer` for the 'upload_image' action. Ensure that this serializer is correctly handling image uploads and that the `upload_image` method is properly secured and tested.